integrations_core_downloader/README.md

```diff
@@ -0,0 +1,3 @@
+# integrations-core-downloader
```

Comment:
Let's detail how this is going to be used, in which version of the agent it will be used, how this package is different from all other packages in this repo, how it will be build as part of the agent and just why we need it and how it works (in toto, tuf, etc...). The more for our open source community friends, the better 😄